slice/Ice/PropertyDict.ice

```diff
@@ -18,7 +18,6 @@ module Ice
 {
     /// A simple collection of properties, represented as a dictionary of key/value pairs. Both key and value are
     /// strings.
-    /// @see PropertiesAdmin#getPropertiesForPrefix
```

Comment:
There is no way for this `@see` to work unfortunately.
It references an operation (can't be forward declared) and `#include`ing the file where `PropertiesAdmin` is defined would be circular.